tests/dragonfly/instance.py

```diff
@@ -233,6 +236,15 @@ def get_logs_from_psutil(self) -> List[str]:
                 rv.append(file.path)
         return rv
 
+    def print_info_logs_to_debug_log(self):
+        logs = self.log_files
+        for log in logs:
+            if "INFO" in log:
+                with open(log) as file:
+                    logging.debug(f"====== LOG name {log} ======")
```

Comment:
nice. small suggestion to make it more catchy, and fun: github actions support terminal coloring. why not use term colors to color background/foreground to allow easier finding this by eye? 

also with the header  - to make it more "lively", easier to find - to add emojis lilke 🪵🪵🪵🪵🪵🪵🪵  (tree log) or something else ?

Reply:
> nice. small suggestion to make it more catchy and fun: github actions support terminal coloring. why not use term colors to color background/foreground to allow easier finding this by eye?

+1, we already do that for `stdout` by replacing log level/date with port/arrow (courtesy of :man_cook: Vlad)

Reply:
falls back to the same coloring vlad introduced. I also replaced the <level/data> with <port/arrow> to be consistent